src/applications/personalization/profile/tests/components/contact-information/ContactInformation.edit-telephone.unit.spec.jsx

```diff
@@ -60,17 +58,19 @@ function editPhoneNumber(
   const editButton = getEditButton(numberName);
   editButton.click();
 
-  const phoneNumberInput = view.getByLabelText(
-    `${numberName} (U.S. numbers only)`,
-    { exact: false },
+  const phoneNumberInput = $(
+    `va-text-input[label^="${numberName}"]`,
+    view.container,
   );
-  const extensionInput = view.getByLabelText('Extension (6 digits maximum)');
+  const extensionInput = $('va-text-input[label^="Extension"]', view.container);
```

Comment:
did we change the label name from `Extension (6 digits maximum)` to `Extension` here?

Reply:
The `[label^="Extension"]` is an [attribute selector](https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors#syntax) that is only looking for the beginning of the label text (kinda like RegExp)